src/tests/system/tests/test_authentication.py

```diff
@@ -97,3 +98,45 @@ def test_authentication__offline_login(client: Client, provider: GenericProvider
 
     assert client.auth.parametrize(method).password(user, correct), "offline login with correct password failed"
     assert not client.auth.parametrize(method).password(user, wrong), "offline login with wrong password succeeded"
+
+
+@pytest.mark.topology(KnownTopology.AD)
+@pytest.mark.ticket(gh=7174)
+@pytest.mark.parametrize("method", ["su", "ssh"])
+@pytest.mark.parametrize("sssd_service_user", ("root", "sssd"))
+@pytest.mark.require(
+    lambda client, sssd_service_user: ((sssd_service_user == "root") or client.features["non-privileged"]),
+    "SSSD was built without support for running under non-root",
+)
+def test_authentication__login_using_email_address(client: Client, ad: AD, method: str, sssd_service_user: str):
+    """
+    :title: Login using user's email address
+    :description:
+        Testing the feature to login using an email address instead of the userid. The email can be anything and does
+        not have to be derived from the domain name an alias can be used. The email should be case insensitive and
+        permit special characters. For example "user_3@, usER_3@" should both work.
```

Comment:
Not sure if this may lead to a wrong interpretation... I had the impression it is missing a statement like "using user_3@ and usER_3@ will lead to retrieve the same user and not different ones". WDYT?

Reply:
I updated the description. This is more accurate.